netket/sampler/metropolis.py

```diff
@@ -249,13 +253,17 @@ def _init_state(sampler, machine, params, key):
             (sampler.n_chains_per_rank, sampler.hilbert.size), dtype=sampler.dtype
         )
 
+        if config.netket_experimental_pjit and jax.device_count() > 1:
+            σ = put_global(σ)
```

Comment:
If we end up rewriting the hilbert spaces in jax  then we can avoid this and instead jit the random state with the correct out_shardings param

Reply:
done